build.gradle

```diff
@@ -137,6 +137,8 @@ project(':forge') {
     apply plugin: 'net.minecraftforge.gradle.patcher'
     apply plugin: 'net.minecrell.licenser'
     apply plugin: 'de.undercouch.download'
+
+    apply from: "$rootDir/gradle/patchVerifer.gradle"
```

Comment:
The better way to do this would be to define the task in buildSrc similar to how we do things in MCPConfig.